framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -0,0 +1,397 @@
+    _value: ATTR_MSG_TYPE
+  data_length:
+    _type: uint16_t
+    _value: 1
+  data:
+    _type: eWscValues8
+    _value: WSC_M1
+
+sWscAttrUuidE:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_UUID_E
+  data_length:
+    _type: uint16_t
+    _value: WSC_UUID_LENGTH
+  data:
+    _type: uint8_t
+    _length: [ WSC_UUID_LENGTH ]
+
+sWscAttrUuidR:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_UUID_E
+  data_length:
+    _type: uint16_t
+    _value: WSC_UUID_LENGTH
+  data:
+    _type: uint8_t
+    _length: [ WSC_UUID_LENGTH ]
+
+sWscAttrMac:
+  _type: struct
+  attribute_type: 
+    _type: eWscAttributes
+    _value: ATTR_MAC_ADDR
+  data_length:
+    _type: uint16_t
+    _value: WSC_MAC_LENGTH
+  data:
+    _type: uint8_t
+    _length: [ WSC_MAC_LENGTH ]
+
+sWscAttrEnroleeNonce:
+  _type: struct
+  attribute_type: 
+    _type: eWscAttributes
+    _value: ATTR_ENROLLEE_NONCE
+  data_length:
+    _type: uint16_t
+    _value: WSC_NONCE_LENGTH
+  data:
+    _type: uint8_t
+    _length: [ WSC_NONCE_LENGTH ]
+
+sWscAttrRegistrarNonce:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_REGISTRAR_NONCE
+  data_length:
+    _type: uint16_t
+    _value: WSC_NONCE_LENGTH
+  data:
+    _type: uint8_t
+    _length: [ WSC_NONCE_LENGTH ]
+
+sWscAttrPublicKey:
+  _type: struct
+  attribute_type: 
+    _type: eWscAttributes
+    _value: ATTR_PUBLIC_KEY
+  data_length:
+    _type: uint16_t
+    _value: WSC_PUBLIC_KEY_LENGTH
+  data:
+    _type: uint8_t
+    _length: [ WSC_PUBLIC_KEY_LENGTH ]
+
+sWscAttrAuthenticationTypeFlags:
+  _type: struct
+  attribute_type: 
+    _type: eWscAttributes
+    _value: ATTR_AUTH_TYPE_FLAGS
+  data_length: 
+    _type: uint16_t
+    _value: 2
+  data: 
+    _type: uint16_t
+
+sWscAttrEncryptionTypeFlags:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_ENCR_TYPE_FLAGS
+  data_length:
+    _type: uint16_t
+    _value: 2
+  data:
+    _type: uint16_t
+
+sWscAttrConnectionTypeFlags:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_CONN_TYPE_FLAGS
+  data_length:
+    _type: uint16_t
+    _value: 1
+  data:
+    _type: eWscConn
+    _value: WSC_CONN_ESS
+
+sWscAttrConfigurationMethods:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_CONFIG_METHODS
+  data_length:
+    _type: uint16_t
+    _value: 2
+  data:
+    _type: uint16_t
+    _value: (WSC_CONFIG_PHY_PUSHBUTTON | WSC_CONFIG_VIRT_PUSHBUTTON)
+
+sWscAttrWscState:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_WSC_STATE
+  data_length:
+    _type: uint16_t
+    _value: 1
+  data:
+    _type: eWscState
+    _value: WSC_STATE_NOT_CONFIGURED
+
+sWscAttrManufacturer:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_MANUFACTURER
+  data_length:
+    _type: uint16_t
+    _value: WSC_MAX_NAME_LENGTH
+  data:
+    _type: char
```

Comment:
Actually it's easier for the programmer to work with chars, since in this case we provide (in tlvf code) simple set method that accepts string, instead of working directly on the buffer and using memcpy (which is the only way with uint8_t buffer). 
Do you think it's critical?
@tomereli @rammarzin what do you think?